cdap-docs/_common/common-build.sh

```diff
@@ -358,7 +367,57 @@ function display_version() {
   echo "PROJECT_SHORT_VERSION: ${PROJECT_SHORT_VERSION}"
   echo "GIT_BRANCH: ${GIT_BRANCH}"
   echo "GIT_BRANCH_TYPE: ${GIT_BRANCH_TYPE}"
-  echo ""
+}
+
+function set_message_file() {
+  TMP_MESSAGE_FILE="/tmp/$(basename $0).$$.tmp"
+  export TMP_MESSAGE_FILE
+}
+
+function clear_message() {
+  MESSAGE=""
+  set_message_file
+}
+
+function set_message() {
+  if [ "${MESSAGE}" == "" ]; then
+    MESSAGE="${1}"
+  else
+    MESSAGE="${MESSAGE}\n\n${1}"
+  fi
+  if [ "x${TMP_MESSAGE_FILE}" != "x" ]; then
+    local warning_message="Warning Message for \"${MANUAL}\":\n${1}"
+    if [ -a ${TMP_MESSAGE_FILE} ]; then
+      echo -e "\n${warning_message}" >> ${TMP_MESSAGE_FILE}
+    else
+      echo -e "${warning_message}" >> ${TMP_MESSAGE_FILE}
+    fi
+  fi
+}
+
+function display_any_messages() {
+  if [ "${MESSAGE}" != "" ]; then
```

Comment:
Do you care if MESSAGE has contents or if it's set? If only contents, use `-n` here.